emcc.py

```diff
@@ -3637,6 +3632,10 @@ def consume_arg_file():
     elif arg.startswith('-o'):
       options.output_file = removeprefix(arg, '-o')
       newargs[i] = ''
+    elif check_arg('-target') or check_arg('--target'):
```

Comment:
Clang doesn't seem to support `--target`, only `-target`. Maybe we should do the same?

Reply:
clang supports both `--target=wasm32` and `-target` `wasm32`

Reply:
oh, and we've already separated out the `=` delimiter here?